### PR TITLE
Added job ad for senior SW engineer, changed backend position

### PR DESCRIPTION
**Pull request by rbordo**
Added job ad for a senior software engineer backend and mobile.
Changed job ad of senior software engineer to a non-senior position.

### CODE REVIEW

File: _posts/jobs/2016-09-30-be-s.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* Understand how to build, release and run a complex software system
+* Understand CD/CI and build pipelines
+* Understand test automation
+* Understand how to scale applications and services
+* Persistence Technologies (SQL, NoSQL)
+
+**Experience in any of the below would be a plus:**
+
+* Spring-Cloud
+* AWS
```
Comment (ghost) [inline _posts/jobs/2016-09-30-be-s.md:38]: feels strange to not mention Android at all if we are looking for a mobile guy

File: _posts/jobs/2016-09-30-be-s.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* You are a team player with experience working in an agile environment
+* You are keen to develop your skills, and those of your team
+* You have an interesting hobby such as brewing beer or space travel
+
+
+You may be young or you may be a veteran in the software development world. This is not important to us, but remember this is a senior position and the bar is set high.
+If this is you, get in touch with us!
+
+ 
+If this is you, [get in touch with us](mailto:jobs@updudes.net)!
```
Comment (ghost) [inline _posts/jobs/2016-09-30-be-s.md]: twice <If this is you,>

**Comment by ghost** [comment]
@rbordo - just this two minor things otherwise looks good to me

File: _posts/jobs/2016-09-30-be-s.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+date: 2017-02-15T12:00:00-04:00
+---
+## About the Role
+You are a Software Engineer by heart. You are experienced in backend and mobile development. Others would call you a full-stack developer. You can distinguish between good and bad software architecture. You can even smell it. You have an architectural vision, you have your point, can convince others but not being too bossy at the same time. It’s easy for you to explain technical stuff to techies as well as to non-techies. You know how to code and you do it often!
+
+We have good engineers, but we need someone who adds the little extra on top.
+
+**Responsibilities:**
+
+* You keep an overview of backend and mobile architecture, adjust if necessary
```
Comment (florina-muntenescu) [inline _posts/jobs/2016-09-30-be-s.md]: what's `adjust if necessary`?

**Review by florina-muntenescu — APPROVED** [review]

File: _posts/jobs/2016-09-30-be-s.md (hunk 1 above)
Comment (rbordo) [reply _posts/jobs/2016-09-30-be-s.md:38]: true, I change it

File: _posts/jobs/2016-09-30-be-s.md (hunk 2 above)
Comment (rbordo) [reply _posts/jobs/2016-09-30-be-s.md]: doppelt gemoppelt ;-)
I change it

File: _posts/jobs/2016-09-30-be-s.md (hunk 3 above)
Comment (rbordo) [reply _posts/jobs/2016-09-30-be-s.md]: adapt?

**Pull request merged**